zipline/transforms/batch_transform.py

```diff
@@ -34,8 +34,12 @@
 
 from zipline.utils.data import MutableIndexRollingPanel
 from zipline.protocol import Event
+from zipline.finance.trading import TradingEnvironment
 
-from zipline.finance import trading
+# HACK the BatchTransform module stores a trading environment to be used by
+# the transforms
+# TODO remove this hack, if not this whole module
+_batch_transform_env = TradingEnvironment()
```

Comment:
I hate having had to do this, but it was the best way to get this module to work gracefully.